charts/quickwit/values.yaml

```diff
@@ -159,7 +260,43 @@ config:
   # searcher:
   #   fast_field_cache_capacity: 10G
   #   split_footer_cache_capacity: 1G
-  #   max_num_concurrent_split_streams: 100  
+  #   max_num_concurrent_split_streams: 100
+
+  indexes: []
+    # - version: 0
+    #   index_id: myindex
+    #   doc_mapping:
+    #     field_mappings:
+    #       - name: project_id
+    #         type: u64
+    #         fast: true
+    #       - name: session_id
+    #         type: u64
+    #         fast: true
+    #     partition_key: project_id
+    #     max_num_partitions: 100000
+    #     tag_fields: [project_id]
+    #   search_settings:
+    #     default_search_fields: [text]
+    #   indexing_settings:
+    #     timestamp_field: view_time
+    #     merge_policy:
+    #       type: limit_merge
+    #       max_merge_ops: 3
+    #       merge_factor: 10
+    #       max_merge_factor: 12
+
+  sources: []
+    # - index: myindex
+    #   source: 
+    #     source_id: mysource
```

Comment:
Currently it's not possible to set a `group-id`, let's remove that parameter.

Something like this is ok:

```
# - index: my-index
    #   source: 
    #     source_id: my-source
    #     source_type: kafka
    #     num_pipelines: 1
    #     params:
    #       topic: quickwit-topic
    #       client_params:
    #         bootstrap.servers: kafka-server-endpoint1:9092,kafka-server-endpoint2:9092
```